packages/firestore/test/integration/api/fields.test.ts

```diff
@@ -398,6 +398,15 @@ apiDescribe('`undefined` properties', (persistence: boolean) => {
     });
   });
 
+  it('are ignored in set({ merge: true })', () => {
+    return withTestDocAndSettings(persistence, settings, async doc => {
+      await doc.set({ foo: 'foo', 'bar': 'unchanged' });
```

Comment:
I am surprised the pre-commit hook didn't drop the quotes around `bar`.

Reply:
Fixed.